### ISSUE TITLE
String concatenation

### ISSUE THREAD

**Issue by czosel** [prettier/plugin-php#152]
Input:
```php
            $this->view->textareaContentId = 'a' . $activationId . 'n' . $noticeTypeId;
```
Output:
```php
            $this->view->textareaContentId = 'a' .
            $activationId .
            'n' .
            $noticeTypeId;
```
I'd suggest
```php
            $this->view->textareaContentId = 
                 'a' . $activationId . 'n' . $noticeTypeId;
```
or (for longer cases)
```php
            $this->view->textareaContentId = 'a' .
                $activationId .
                'n' .
                $noticeTypeId;
```

**Comment by czosel** [comment]
It seems that this needs to be fixed inside `printBinaryExpression` - @mgrip, could you take a look?

**Comment by mgrip** [comment]
Will do!

**Comment by mgrip** [comment]
So I think this is actually related to how we print `assign` nodes - here's how the JS printer handles it: https://github.com/prettier/prettier/blob/master/src/language-js/printer-estree.js#L4807

**Comment by mgrip** [comment]
Fixed in #177
